repository/src/test/java/org/alfresco/repo/content/ContentServiceImplUnitTest.java

```diff
@@ -142,6 +145,21 @@ public void testRequestContentDirectUrl_SystemWideIsDisabled()
         }
     }
 
+    @Test
+    public void testRequestContentDirectUrlwithQname_SystemWideIsDisabled()
+    {
+        setupSystemWideDirectAccessConfig(DISABLED);
+        try
+        {
+            contentService.requestContentDirectUrl(NODE_REF, QNAME, true, 20L);
+            fail("Expected DirectAccessUrlDisabledException");
+        }
+        catch (DirectAccessUrlDisabledException ex)
+        {
+            verify(mockContentStore, never()).isContentDirectUrlEnabled();
+        }
+    }
```

Comment:
Not needed. Just update the method signature in the old test. Once deprecated the old signature should eventually be removed from the product.

Reply:
I would keep the tests - once the deprecated methods are removed - these test of course need to be removed as well.